### ISSUE TITLE
Fails to extract an email when there is a "at" word before it 

### ISSUE THREAD

**Issue by webrdaniel** [gajus/extract-email-address#7]
Fails to extract an email when there is a "at" word before it 
e.g. these two emails in text: "Additionally, feel free to contact us at z.test@apify.com or at jost.sur123@domain.org."
Note that the email are containing "." in the first part.

**Comment by gajus** [comment]
Yeah, the problem here is that the string is first normalized to ~

```diff
-or at jost.sur123@domain.org
+or@jost.sur123@domain.org
```

The fix would be to detect when `at` is followed by a string with `@` and ignore that scenario. Would accept a PR.

**Comment by gajus** [comment]
Fixed
